File: Python/Card.py
from Payment import Payment

class Card(Payment):
    numberCard = int
    dataCard = str
    cvv = str

    def __init__(self, id, numberCard, dataCard, cvv):
        super().__init__(id)
        self.numberCard = numberCard
        self.dataCard = dataCard
        self.cvv = cvv

File: Java/Card.java
public class Card extends Payment{
    Integer numberCard;
    String dataCard;
    String cvv;

    public Card(Integer id, Integer numberCard, String dataCard, String cvv){
        super(id);
        this.numberCard = numberCard;
        this.dataCard = dataCard;
        this.cvv = cvv;
    }
}
